packages/client/mutations/UpsertTeamPromptResponseMutation.ts

```diff
@@ -59,33 +66,9 @@ const UpsertTeamPromptResponseMutation: StandardMutation<TUpsertTeamPromptRespon
   return commitMutation<TUpsertTeamPromptResponseMutation>(atmosphere, {
     mutation,
     variables,
-    optimisticUpdater: (store) => {
-      const {viewerId} = atmosphere
-      const {meetingId, content} = variables
-      const meeting = store.get(meetingId)
-      if (!meeting) return
-      const phases = meeting.getLinkedRecords('phases')
-      if (!phases) return
-      const stages = phases[0]!.getLinkedRecords('stages')
-      if (!stages) return
-      const stage = stages.find(
-        (s) => s.getLinkedRecord('teamMember')?.getValue('userId') === viewerId
-      )
-      if (!stage) return
-      const response = stage.getLinkedRecord('response')
-      if (!response) {
-        // insertion
-        const newResponse = createProxyRecord(store, 'TeamPromptResponse', {
-          id: clientTempId(viewerId),
-          meetingId,
-          sortOrder: 0,
-          content
-        })
-        stage.setLinkedRecord(newResponse, 'response')
-      } else {
-        // update
-        response.setValue(content, 'content')
-      }
```

Comment:
we don't need an optimistic updater at all. relay store will be updated upon the successful finish of mutation, tip tap always displays the actual content so an optimistic updater doesn't make any difference. if I miss something, I'm happy to improve.

Reply:
-1 It seems like the optimistic updater is needed for cases where content changes between null and non-null.  Specifically, there seems to be a bug where the editor state flashes between the old and new state in these cases.  See loom - https://www.loom.com/share/52896ca850d64a57b5ad3bbc20342cee

I think this will also be needed to immediately update the response in an open discussion drawer in https://github.com/ParabolInc/parabol/pull/6469/commits/1bdb26fd6af9b2eb525473490c5ce3313cd5de24#diff-b3416557ebc46ac9b7d1fa9ef62f5564cf40767ea9895e098715c22cb32d4549R68

Reply:
@jmtaber129 oh, that's correct. I've added the `optimisitcUpdater`, could please double-check now? thank you!